components/event-sources/cmd/http-adapter/main.go

```diff
@@ -97,27 +91,14 @@ func setupAdapter(component string, ector adapter.EnvConfigConstructor, ctor ada
 		logger.Error("error building statsreporter", zap.Error(err))
 	}
 
-	tracingCfg := DisabledTracingConfig
-
-	switch v := interface{}(ector).(type) {
-	case eshttp.AdapterEnvConfigAccessor:
-		if v.IsTracingEnabled() {
-			tracingCfg = tracing.OnePercentSampling
-		}
-	}
-	if err = tracing.SetupStaticPublishing(logger, "", tracingCfg); err != nil {
-		// If tracing doesn't work, we will log an error, but allow the adapter
-		// to continue to start.
-		logger.Error("Error setting up trace publishing", zap.Error(err))
-	}
-
-	eventsClient, err := kncloudevents.NewDefaultClient(env.GetSinkURI())
+	v := env.(eshttp.AdapterEnvConfigAccessor)
+	ceClient, err := eshttp.NewCloudEventsClient(v.GetPort())
 	if err != nil {
-		logger.Fatal("error building cloud event client", zap.Error(err))
+		logger.Fatalf("Could not create cloudevents client: %+v", err)
```

Comment:
Can you explain why you think so? It is not an error which can get further included in a message (it will print and do os.exit), so upper-case is fine IMHO. What do you think?

Reply:
This is because of logger.Fatal() and logger.Error(), both use 
```
s.log(with_right_level, "", args, nil)
```
internally and usually printing the message with context(the reason of fatal or error).

Hence the logic of the "case" should be similar, IMO.
Reference: [code review comments](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)

Reply:
As discussed internally:

> This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.

The "Could..." is not combined inside other message. So it is fine.

```go
logger2 := logger.Sugar()
logger2.Fatalf("Foo: %+v", errors.New("foo"))

2020-08-11T14:01:55.238+0200    FATAL   cmd/slugify.go:22       Foo: foo
main.main
        /Users/i512777/go-working/src/github.com/nachtmaar/slug/cmd/slugify.go:22
runtime.main
        /usr/local/Cellar/go/1.14.5/libexec/src/runtime/proc.go:203
exit status 1
```

However, accidentally `log.Fatal` with `zap.Error` was not used. Changing it.